apps/editors/decorators.py

```diff
@@ -33,6 +33,23 @@ def wrapper(request, *args, **kw):
     return wrapper
 
 
+def unlisted_addons_reviewer_required(f):
+    """Requires the user to be logged in as an unlisted addons reviewer or
+    admin.
+
+    An unlisted addons reviewer is someone who is in a group with the following
+    permission: Addons:ReviewUnlisted.
+
```

Comment:
Extra blank link, and missing newline after opening quotes. I know the other decorators are like this as well, but it makes me twitch.